reactjs/src/pages/blocks/leftnav.js

```diff
@@ -17,6 +62,13 @@ export default function LeftNav() {
                         <NavLink to={link.path}>{link.pageName}</NavLink>
                     </li>
                 ))}
+
+                <div style={{ position: 'relative', width: '600px', height: '800px' }}>    
```

Comment:
Why are we setting width to 600px here when the left nav (parent) is only 2 columns wide?  Why aren't we using col-md-12?  Why are we setting the height to 800px as well?  Is there not something you can use that's already built into the bootstrap classes?